### DISCUSSION TITLE
API REST do GITHUB - Dúvida na lista de contribuintes do respositório

### DISCUSSION THREAD

**Discussion by gabrieldeoliveiraestevam** [community/community#24749] [Apps, API and Webhooks]
<p>Guys,</p>
<p>I’m using a GITHUB REST API to list contributors to a repository.</p>
<p>I took the repository as an example: FortAwesome/Font-Awesome</p>
<p>When accessing the general page of the repository it appears that there are 93 contributors. And when I enter the contributors link, there are only 7 contributors. API returns me 7 contributors.</p>
<p><strong>Can you explain that to me? Why in one place does it say that the repository has 93 contributors and in another there are only 7?</strong></p>
<p>Repository link - 93 contributors: <a href="https://github.com/FortAwesome/Font-Awesome" class="inline-onebox" rel="noopener nofollow ugc">GitHub - FortAwesome/Font-Awesome: The iconic SVG, font, and CSS toolkit</a></p>
<p>Contributors page link - 7 contributors: <a href="https://github.com/FortAwesome/Font-Awesome/graphs/contributors" class="inline-onebox" rel="noopener nofollow ugc">Contributors to FortAwesome/Font-Awesome · GitHub</a></p>

**Comment by parithiban** [comment, marked as answer]
<p><a class="mention" href="/gabrieldeoliveiraestevam">@gabrieldeoliveiraestevam</a></p>
<p>From my understanding a contributor may be who</p>
<p>1.Created a commit<br>
2.Created an issue<br>
3.Created a pull request<br>
4.Created the repository<br>
5.Reviewed a pull request</p>
<p><a href="https://github.blog/2013-01-07-introducing-contributions/#repositories-contributed-to" rel="noopener nofollow ugc">Blog Reference</a></p>
<p>This is why it is showing <code>93</code> in the Repo page.</p>
<p>To explain about the Contributors page link(<code>7 Contributor</code>) as you can see in the reference image it will just show <code>Contributions commited to master branch</code></p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/3/5/353ca41dd431a855c3540fe42bec3c7ec5788611.png" data-download-href="/uploads/short-url/7AXebH5bAtuhPXLkMhj0AdX85K9.png?dl=1" title="Screenshot 2021-05-30 at 8.07.30 PM" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/7281166/181086567-170983a9-269e-4ffa-9ea7-e6343de6992a.png" alt="Screenshot 2021-05-30 at 8.07.30 PM" data-base62-sha1="7AXebH5bAtuhPXLkMhj0AdX85K9" width="690" height="315" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/3/5/353ca41dd431a855c3540fe42bec3c7ec5788611_2_690x315.png, https://aws1.discourse-cdn.com/github/optimized/3X/3/5/353ca41dd431a855c3540fe42bec3c7ec5788611_2_1035x472.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/3/5/353ca41dd431a855c3540fe42bec3c7ec5788611_2_1380x630.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/3/5/353ca41dd431a855c3540fe42bec3c7ec5788611_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screenshot 2021-05-30 at 8.07.30 PM</span><span class="informations">2880×1316 326 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div><br>
`.

**Comment by gabrieldeoliveiraestevam** [comment]
<p>thanks. Now I get it.</p>
